File: projectEuler/p1.c
#include <stdio.h>

/**
 * main - Problem 1
 *
 * Description: Multiples of 3 or 5
 *
 * Return: Always 0 (Success)
*/

int main(void)
{

	int sum = 0;

	for (int i = 0; i < 1000; ++i)
	{
		if (i % 3 == 0 || i % 5 == 0)
			sum += i;
	}

	printf("The sum of all the multiples of 3 or 5 below 1000 is %d\n", sum);

	return (0);
}

File: projectEuler/p1.py
#!/usr/bin/env python3

"""
problem 1; Multiples of 3 or 5
"""

sum = 0

for i in range(1000):
    if (i % 3 == 0) or (i % 5 == 0):
        sum += i

print(f"The sum of all the multiples of 3 or 5 below 1000 is {sum}")
